tests/drop_counters/test_drop_counters.py

```diff
@@ -29,10 +29,21 @@
 def pkt_fields(duthost):
     # Gather ansible facts
     mg_facts = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
+    ipv4_addr = None
+    ipv6_addr = None
+
+    for item in mg_facts["minigraph_bgp"]:
+        if item["name"] == mg_facts["minigraph_bgp"][0]["name"]:
```

Comment:
Because mg_facts["minigraph_bgp"] is a list of dictionary. And it should find ipv6 and ipv4 bgp info for the same VM. IPv4 and IPv6 are separate list items.

Reply:
It looks to me that this code is identical to:

```python
item = mg_facts["minigraph_bgp"][0]
if "." in item["addr"]:
    ipv4_addr = item["addr"]
else:
    ipv6_addr = item["addr"]
```

Reply:
I see